### DISCUSSION TITLE
Account Privacy Settings crash the app

### DISCUSSION THREAD

**Discussion by Akimayo** [pixelfed/pixelfed-rn#45] [Bug Reports]
Opening **Settings** > **Privacy & Relationships** > **Account Privacy** loads for a bit and then crashes the app.

Running Pixelfed v1.0.0.45 (from APK) on Android 13, instance [pixey.org](https://pixey.org).

**Comment by dansup** [comment]
This means the instance needs to update, this has been fixed by restricting servers to a min version of 0.12.1 when logging in.
